pwiz_tools/BiblioSpec/src/DiaNNSpecLibReader.cpp

```diff
@@ -622,9 +628,9 @@ bool DiaNNSpecLibReader::parseFile()
             if (kvp.first < 1)
                 break;
 
-            io::CSVReader<1, io::trim_chars<' ', ' '>, io::no_quote_escape<'\t'>> reportReader(kvp.second.string().c_str());
-            reportReader.read_header(io::ignore_extra_column | io::ignore_missing_column, "Precursor.Id");
```

Comment:
Why read "Run" here if it not checking for it below in `if (reportReader.has_column`?

Reply:
I was thinkinkg forward to when I would test for "Run" being absent and "Lib.1/K0" would be present instead of "IMS" and this would indicate Bruker Paser results. Though, I am not totally sure they want to provide .speclib file at all yet instead of just a collection of .tsv files. So, maybe you are right and we should remove "Run" here.